### ISSUE TITLE
How to interpret the training result?

### ISSUE THREAD

**Issue by SHKChan** [ultralytics/yolov5#1831]
## ❔Question
Hi, are there any tutorial can tell me how to interpret the training result? I mean like what the differences between obj_loss and box_loss, and so on. And how to tune my network or other setting base on those result like mAP and Percsion. Sorry just new to neural network and object detection. Thanks for you guys help!

**Comment by glenn-jocher** [comment]
@SHK2018 the original YOLOv3 paper has a pretty good explanation of the loss components:
https://pjreddie.com/publications/

For metrics you may want to visit:
https://en.wikipedia.org/wiki/Precision_and_recall

**Comment by SHKChan** [comment]
Thanks! Really helpful for me. I will close this issue.

**Comment by MERTZGZ** [comment]
> @SHK2018 the original YOLOv3 paper has a pretty good explanation of the loss components: https://pjreddie.com/publications/
> 
> For metrics you may want to visit: https://en.wikipedia.org/wiki/Precision_and_recall

https://pjreddie.com/publications/  
i can't access the information you said on this link, please check it again.

**Comment by glenn-jocher** [comment]
@MERTZGZ link is correct

<img width="994" alt="Screen Shot 2021-12-28 at 11 04 15 AM" src="https://user-images.githubusercontent.com/26833433/147598656-6a8d4d96-eea6-46eb-9dfa-6b99d88336dd.png">
